clearblade.md

```diff
@@ -1388,6 +1388,118 @@ Returns the current topics for a system
     msg.getAndDeleteMessageHistory("TestTopic", 0, null, null, null, callback); // get and delete all messages for "TestTopic"
 ~~~
 
+## messaging.setTimeout(timeout, topic, data, callback)
+
+This method publishes data to a topic after timing out.
+
+Callback setTimeout:
+
+ * @param {integer} timeout - Timeout in miliseconds
+ * @param {string} topic - String that signifies which topic to search
+ * @param {string} data - response -messages
+ * @param {function} callback - Function that handles the response from the server
```

Comment:
Can we have callback something like this: https://jsdoc.app/tags-callback.html
Ex: 
`* @callback  cbSetTimeout
* @param {boolean} err - Is true if there is an error
* @param {string} msg - timerId if no error, error message otherwise`

`function setTimeout:
  * @param {integer} timeout - Timeout in miliseconds
 * @param {string} topic - String that signifies which topic to search
 * @param {string} data - response -messages
 * @param {cbSetTimeout} callback - Function that handles the response from the server`

Reply:
This applies to all the method you added. I am aware other older methods do not follow the same convention, but we can do nothing much about it really.